### ISSUE TITLE
Feature Request - Analogue Stick/Trackball for Vector Games

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kexplo/imame4all#5]
```
Hi

Fantastic App. Really well done.

Do you think you could add support for an analogue joystick so that games like 
Star Wars are playable?

Also, could you emulate a trackball - for games like tempest or missile command?

Thanks for listening
```

Original issue reported on code.google.com by `z...@zoesummers.com` on 1 Sep 2010 at 10:36

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I was wrong. It will be added on next update.

```

Original comment by `seleuco....@gmail.com` on 9 Sep 2010 at 10:56

**Comment by GoogleCodeExporter** [comment]
```
Fixed on 1.4
```

Original comment by `seleuco....@gmail.com` on 15 Sep 2010 at 5:20
- Changed state: **Fixed**
